utils/markdown/autolink.go

```diff
@@ -14,6 +14,7 @@ import (
 
 var (
 	DefaultUrlSchemes = []string{"http", "https", "ftp", "mailto", "tel"}
+	wwwAutoLinkRegex  = regexp.MustCompile(`^www\d{0,3}\.`)
```

Comment:
The need for this regexp escapes me. Why do we look for up to 3 digits after `www`?

Reply:
That's a good point. Maybe @hmhealey can shed some light.

Reply:
I forget why specifically we added it, but I know I've seen websites hosted at things like `www1.example.com`.